### DISCUSSION TITLE
Text Gradient Color : WCAG 1.4.3

### DISCUSSION THREAD

**Discussion by Ignacio-de-Loyola** [w3c/wcag#3715] [Q&A]
![image](https://github.com/w3c/wcag/assets/80868256/e4693177-08b7-4635-bc58-eca5ba170810)
Hi Team,

A gradient is applied to the tabs on the left and right that are not activated. It's not activated so the text isn't completely visible, but you can still see part of the text. 
In this case, does the text with the gradient applied have to be at least 4.5:1 to comply with the conditions of WCAG 1.4.3?

**Comment by JAWS-test** [comment]
My guess is: The text is not readable, so it is irrelevant. The only thing that is relevant is that it is recognisable that there are further tabs on the right and left, i.e. there must be at least one graphic element with a minimum contrast of 3:1 with which I can recognise that there are tabs. Of course, it would be ideal if part of the visible text had a contrast of 4.5:1
